docs/content/engines/presto.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Presto"
+weight: 5
+type: docs
+aliases:
+- /engines/presto.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+This documentation is a guide for using Paimon in Presto.
+
+## Version
+
+Paimon currently supports Presto 0.236 and above.
+
+## Deploy Paimon Presto Connector
+
+Build bundled jar with the following command.
+
+|     Version      | Command                                                                                                                                                                                                            |
+|------------------|-------------------------------------------------------------------------|
+| [0.236,0.268)    | mvn clean install -Dmaven.test.skip=true -Ppresto-0.236                 |
```

Comment:
Add download jars like spark 3?

Reply:
Thanks for your review，I will do this later.